examples/ensemble/plot_random_forest_embedding.py

```diff
@@ -38,9 +37,7 @@
 
 # use RandomTreesEmbedding to transform data
 hasher = RandomTreesEmbedding(n_estimators=10, random_state=0, max_depth=3)
-hasher.fit(X)
-model = SelectFromModel(hasher, prefit=True)
-X_transformed = model.transform(X)
```

Comment:
Same remark here: it seems that these changes are unrelated to Arpack initialization. I don't understand.